config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
-  post "/", to: "graphql#execute"
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  post '/', to: 'graphql#execute'
+  # graphQL IDE
+  if Rails.env.development?
+    mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "graphql#execute"
+    post "/graphql", to: "graphql#execute"
```

Comment:
Set this up so that in development, the interface expects this endpoint but we were using the other in our tests/production, so this allows us to use both

Reply:
This endpoint will only be activated in development

Reply:
Awesome flexibility here and makes use of a great development tool